cogs/link_fixer.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import re
+from urllib.parse import urlparse
+
+from discord.ext import commands
+
+logger = logging.getLogger(__name__)
+
+
+class LinkFixerCog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.embed_domains = {
```

Comment:
You can read these from a file instead of initializing in code. Will let us update stuff without a code change. Can be fancy and read every X mins/hours so it doesn't even require a new deployment

Reply:
Good idea, however it will still require cog reloading of sorts, and added complexity. Maybe one for an improvement after.

Reply:
wonder if this can work https://github.com/robertwayne/cogwatch 
@ingenium21 i am ok shipping this as-is, but if its low effort to integrate with a library like ^, we might as well add it (or at least create a separate GitHub issue)

Reply:
yeah lets turn this into a separate issue.  Besides i don't know how long this dict is going to get, but if it does, we can switch to a json.

Reply:
for now can you approve so we can push this forward?

Reply:
@ash663 can you approve and I can pinky promise to add cogwatch to my next PR

Reply:
mb, i thought you had two approvals already

Reply:
i thought so too but for some reason V's didn't count? his approval is grey